lib/workers/repository/onboarding/pr/index.ts

```diff
@@ -152,14 +152,16 @@ If you need any further assistance then you can also [request help here](${confi
       logger.info({ pr: pr.displayNumber }, 'Onboarding PR created');
       await addParticipants(config, pr);
     }
-  } catch (err) /* istanbul ignore next */ {
+  } catch (err) {
     if (
-      err.statusCode === 422 &&
+      err.response?.statusCode === 422 &&
```

Comment:
`err.statusCode` invokes a getter for `err.response.statusCode`

tested on real repo:
```
DEBUG: getPrList() (repository=ladzaretti/onboarding)
DEBUG: Creating onboarding PR (repository=ladzaretti/onboarding)
DEBUG: Creating PR (repository=ladzaretti/onboarding)
       "title": "Configure Renovate",
       "head": "ladzaretti:renovate/configure",
       "base": "main",
       "draft": false
 WARN: Onboarding PR already exists but cannot find it. It was probably created by a different user. (repository=ladzare
tti/onboarding)

```